backend/LearningCalendar/Epicenter.Api.Service.Interface/Operations/Goal/IGetPersonalGoalsOperation.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+
+namespace Epicenter.Service.Interface.Operations.Goal
+{
+    public interface IGetPersonalGoalsOperation
+    {
+        Task<GetPersonalGoalsOperationResponse> Execute();
+    }
+
+    public class GetPersonalGoalsOperationResponse
+    {
+        public List<PersonalGoal> PersonalGoals { get; set; }
+
+        public class PersonalGoal
+        {
+            public Guid Id { get; set; }
```

Comment:
just in case we decide to change the Completion Status logic for goals, perhaps add a bool IsCompleted => CompletionDate.HasValue